swan-lake/development-tutorials/source-code-dependencies/style-guide/top-level-definitions.md

```diff
@@ -50,6 +50,50 @@ intro: The sections below include the coding conventions with respect to top-lev
 
 * Imports should be sorted alphabetically, first by the organization name and then by the module name.
 
+## Constant declaration
+
+* Use the SCREAMING_SNAKE_CASE for constant names.
+
+    **Example,**
```

Comment:
The page has uses both `:` and `,` after `Example`. The correct use of punctuation is the `:`.
```suggestion
    **Example:**
```
![image](https://github.com/ballerina-platform/ballerina-dev-website/assets/7709843/3398feb5-dd74-4782-a03c-c929674895be)

Reply:
Updated all places on the same page.